tests/expected/yoda-compairson.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
you forgot to fix the typo on filename

Reply:
Codesniffer should fix these kind of typos too :smile: